rules/linux/process_creation/proc_creation_lnx_rev_shell_ruby.yml

```diff
@@ -0,0 +1,26 @@
+title: Possible Ruby Reverse Shell
+id: b8bdac18-c06e-4016-ac30-221553e74f59
+status: experimental
+description: Detects some of most common reverse shell techniques on Linux systems.
+references:
+      - https://pentestmonkey.net/cheat-sheet/shells/reverse-shell-cheat-sheet
+      - https://www.revshells.com/
+author: '@d4ns4n_'
+date: 2023/04/07
+tags:
+   - attack.initial_access
+   - attack.t1133
+logsource:
+   category: process_creation
+   product: linux
+detection:
+    selection:
+        Image|contains: 'ruby'
+        CommandLine|contains|all:
+            - '-rsocket'
+            - '-e'
```

Comment:
here too

Reply:
Hi @Neo23x0 for Ruby I think the right match is `-e'`.
In the commit I have set '`-e\'`' to escape the " ' ".